hacep/src/test/java/org/kie/hacep/consumer/CommandHandlerTest.java

```diff
@@ -265,6 +283,28 @@ public void visitSnapshotOnDemandCommand() {
                          });
     }
 
+    @Test
+    public void visitGetKJarGAVCommand() {
+        executeAndVerifyResponseMessage(new GetKJarGAVCommand(myEntryPoint),
+                                        commandHandler::visit,
+                                        GetKJarGAVMessage.class,
+                                        result -> kJarGAV.equals(result));
+    }
+
+    @Test
+    public void visitUpdateKJarCommand() {
```

Comment:
Please add more unit tests. Add some test which will use some real kjar to be imported and send the command using Kafka.

Reply:
A Integration test can't be executed because the settings.xml from the user's home folder is used by the Aether used in the Drools builder, this mean change the settings.xml manually on the host machine before the tests.

Reply:
There must be a possibility to develop integration tests. We have quite a similar scenario in execution server and it is covered by tests. As far as I know it uses local Maven repository.

Reply:
With Maven we could pass an alternative settings.xml but here we have only Aether used by Drools and no arguments to send.